drivers/sensor/lis2dw12/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+zephyr_library()
```

Comment:
Add license/copyright header

Reply:
Very few CMakelist.txt have the header, but I agree that they should.
I'll add it also to LPS22HH, which is the other sensor driver I PR-ed recently

Reply:
Thanks!